spec/controllers/posts_controller_spec.rb

```diff
@@ -0,0 +1,20 @@
+require 'rails_helper'
+
+describe PostsController do
+
+  describe 'GET index' do
+    it 'assigns all posts as @posts' do
+      post = FactoryGirl.create(:post)
+      get :index, {}
+      expect(assigns(:posts)).to eq([post])
+    end
+  end
+
+  describe 'GET show' do
+    it 'assigns the requested post as @post' do
+      post = FactoryGirl.create(:post)
+      get :show, {:id => post.to_param}
```

Comment:
Redundant curly braces around a hash parameter.<br>Use the new Ruby 1.9 hash syntax.<br>Space inside { missing.<br>Space inside } missing.